pages/pet-detail-page/[id].js

```diff
@@ -80,9 +80,9 @@ const StyledInteractionImage = styled(Image)`
   top: 0;
   right: 30%;
   animation: ${(props) => {
-      if (props.animationStyle === "sleeping") return sleepingAnimation;
-      if (props.animationStyle === "toy") return toyAnimation;
-      if (props.animationStyle === "food") return foodAnimation;
+      if (props.$animationstyle === "sleeping") return sleepingAnimation;
+      if (props.$animationstyle === "toy") return toyAnimation;
+      if (props.$animationstyle === "food") return foodAnimation;
```

Comment:
Hi, styled components should contain as less logic as possible. 
A function like this could just pass the right prop to the component:

function getAnimationStyle (interaction) {
     if (interaction === "sleeping") {
          return "sleepingAnimation"
    }
etc...
}

And then replace 416 with:
animationStyle={getAnimationStyle(isInteracting.interaction)}

Reply:
Thanks for your review, we decided against this approach in the team for reasons of readability.